From dfb112fead759af935db36f620da30f96b87dcd2 Mon Sep 17 00:00:00 2001
From: jinghua <xigu@marvell.com>
Date: Wed, 4 Jan 2017 00:16:39 +0800
Subject: [PATCH 0745/1345] doc: mvebu: add missing drivers to porting-guide

commit  f6bf483af6e747f779ff0b1259f1a3152518a1c1 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

- driver ehci-orion and dma/mv-xor.txt are used by Armada-xp,
  Armada-370, Armada-380, Armada-390 and Armada3700.
- rest of the drivers in this patch are for Armada3700 only.

Change-Id: Ie0acf38761daed5ebd6acc65b0bcf9b91eb17818
Signed-off-by: jinghua <xigu@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/35309
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
Reviewed-by: Wilson Ding <dingwei@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 Documentation/mvebu/porting_guide.txt |    7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/Documentation/mvebu/porting_guide.txt b/Documentation/mvebu/porting_guide.txt
index 4941e85..4cc5571 100644
--- a/Documentation/mvebu/porting_guide.txt
+++ b/Documentation/mvebu/porting_guide.txt
@@ -83,6 +83,8 @@ Common Drivers
 	Marvell Xenon SDHCI driver	| mmc/marvell,xenon-sdhci.txt
 	Marvell MDIO ethernet driver	| net/marvell-orion-mdio.txt
 	Marvell PCIe driver		| pci/armada8k-pcie.txt
+	Marvell eHCI driver		| usb/ehci-orion.txt
+	Marvell XOR engines driver	| dma/mv-xor.txt
 
 A80x0/A70x0
 ----------
@@ -105,6 +107,11 @@ A37x0
 	Marvell Neta Ethernet controller| net/marvell-armada-370-neta.txt
 	Marvell A37x0 i2c driver	| i2c/i2c-pxa.txt
 	Marvell A37x0 SPI driver	| spi/spi-armada-3700.txt
+	Marvell A37x0 UART driver	| tty/serial/mvebu-uart.txt
+	Marvell A37x0 Clock driver	| clock/mvebu-armada3700-clock.txt
+	Marvell A37x0 GPIO driver	| gpio/gpio-armada-3700.txt
+	Marvell A37x0 PCIe driver	| pci/aardvark-pci.txt
+	Marvell A37x0 USB OTG PHY driver| usb/armada3700-otg-phy.txt
 
 A39x
 ----
-- 
1.7.9.5

